### PR TITLE
#1 #2 #3 #4 #5   updates

### PR DESCRIPTION
**Pull request by maxsiz**

### CODE REVIEW

File: contracts/EventStorage.sol [hunk 1]
```diff
@@ -28,40 +32,51 @@ contract EventStorage {
 
     constructor (string memory _version) {
         version = _version;
+        INSULINS.push(Insulin('Novorapid', 10800 ));
```
Comment (svanas) [inline contracts/EventStorage.sol:35]: NovoRapid duration is 3 hours (aka 180 minutes)

File: contracts/EventStorage.sol [hunk 2]
```diff
@@ -28,40 +32,51 @@ contract EventStorage {
...
+        INSULINS.push(Insulin('Humuline',  18000 ));
```
Comment (svanas) [inline contracts/EventStorage.sol:36]: Humuline duration is 5 hours (aka 300 minutes)

File: contracts/EventStorage.sol [hunk 3]
```diff
@@ -28,40 +32,51 @@ contract EventStorage {
...
-        emit MeasureRecord (
-            msg.sender,
-            _timestamp,
-            _glucose,
-            _carbs,
-            _units,
-            _insulinType
-        );
-
+        if  (personalInsulins[msg.sender].length == 0) {
```
Comment (svanas) [inline contracts/EventStorage.sol:52]: What if I have added at least one personal insulin? I get the impression the hardcoded (built-in) insulins are removed from view.

Comment (maxsiz) [reply contracts/EventStorage.sol:52]: Yes you are right. We can use only one of two arrays: common harcoded or personal

**Review by maxsiz — COMMENTED** [review]
ok

**Pull request merged**